content/departments/product-engineering/engineering/process/releases/index.md

```diff
@@ -135,3 +242,12 @@ This is because:
 - We haven't built the automated testing and update infrastructure to make continuous customer releases reliable.
```

Comment:
Is this true? I think it's actually more nuanced than this. Our customers usually don't have the capacity to upgrade continuously. It can take a customer months (or years) to get to the next incremental release depending on their internal processes. A lot of our customers have rigorous change controls in-place.

I agree that our automated testing could use some work, but definitely think the issue at hand is a little more complicated 😆 

I know you didn't make this change, just saw it and felt something about this line!

Reply:
I am a bit confused.

Are you saying we do have some `automated testing and update infrastructure` for releases? 

About enabling customers to continuously update, the biggest blocker is 

> We don't think customers would be comfortable with a continuously updated service running on their own infrastructure, for security and stability reasons.

If we throw this away, and we do want to CD for on-prem. I saw Dan is talking about giving [Replicated](https://www.replicated.com/) another try. This could potentially allow us to have an automated release process for customers.